lib/executor/vu_handle.go

```diff
@@ -41,7 +42,11 @@ type vuHandle struct {
 	returnVU  func(lib.InitializedVU)
 	config    *BaseConfig
 
+	vu           lib.InitializedVU
 	canStartIter chan struct{}
+	// This is here only to signal that something has changed it must be added to and read with atomics
+	// and helps to skip checking all the contexts and channels all the time
```

Comment:
Maybe it helps to clarify:

```suggestion
	// If change is not 0, it signals that the VU needs to be reinitialized. It must be added to and read
	// with atomics and helps to skip checking all the contexts and channels all the time.
```

Reply:
resolved in 02e96f6